lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -169,6 +179,20 @@ class ReportUIFeatures {
     this._templateContext = context;
   }
 
+  /**
+   * @param {{name: string, text: string, onClick: () => void}} opts
+   */
+  createIconButton(opts) {
+    const metricsEl = this._dom.find('.lh-audit-group--metrics', this._document);
+    const buttonEl =
+      this._dom.createChildOf(metricsEl, 'div', `icon-button icon-button--${opts.name}`);
```

Comment:
we gotta [Just use a button](https://www.youtube.com/watch?v=CZGqnp06DnI)

tldr:  `<button type=button`>